src/sentry/static/sentry/app/views/stream/actions.jsx

```diff
@@ -58,12 +59,21 @@ const getConfirm = (numIssues, allInQuerySelected, query, queryCount) => {
         <p style={{marginBottom: '20px'}}>
           <strong>{question}</strong>
         </p>
-        <ExtraDescription
-          all={allInQuerySelected}
-          query={query}
-          queryCount={queryCount}
-        />
-        {!canBeUndone && <p>{t('This action cannot be undone.')}</p>}
+        <ExtraDescription all={allInQuerySelected} query={query} />
+        {!canBeUndone && (
+          <p>
+            'This action cannot be undone. You may consider '
```

Comment:
This is going to render with quotes, I think?

We also use `t()` or `tct()` with string literals so that we can extract them during our frontend build process so that they can potentially be translated.

try:

```
<p>
{tct('This action cannot be undone. You may consider [link:other alternatives].', {
  link: <Link to="https://help.sentry.io/hc/en-us/articles/360003443113-When-should-I-delete-events-" />

})}
</p>
```